docs/source/tutorial/circular_patch_simple.rst

```diff
@@ -482,6 +482,23 @@ the interpolation is desired.  This can also be done with the constructor as::
 
     interp = Interpolator(list(parrays.values()), x=x, y=y, z=z)
 
+There are some cases, where one may require a higher order interpolation or
+gradient approximation of the property. This can be done by passing a
+``method`` for interpolation to the interplator as::
+
+    interp = Interpolator(list(parrays.values()), num_points=10000, method='order1')
+
+Currently, PySPH have three method of interpolation namely ``shepard``,
```

Comment:
have three method of ... -> has three methods of ...

Reply:
done